### ISSUE TITLE
[3.4.1] Imported two color images slope too much

### ISSUE THREAD

**Issue by rallant92** [Ultimaker/Cura#4542]
**Application Version**
3.4.1

**Platform**
Windows 10 64bit

**Printer**
None

**Steps to Reproduce**
1. Start Cura
2. Import a picture that is a two color silhouette.  A great example is the gear silhouettes at https://www.deviantart.com/dinn/art/Gears-JPG-189806582.  Drag the image to Cura.
3. Fix the import to be "Darker is higher".
4. Select the OK button in the dialog to import.

**Actual Results**
The gears all have sloped walls.  The top layer is much "smaller" than the bottom layer from the image.  Two such gears could never rotate with each other.

**Expected results**
The gears need to have vertically straight walls to work as gears.  The bottom and top layers should be identical.

**Additional Information**
This is also a problem when putting text into the image.  The text becomes distorted and unreadable as the material erodes away with the height.  Essentially images can never be customized with people's names and such.

**Comment by diegopradogesto** [comment]
Yes, this is a known behavior since we don't distinguish between only black and white images with sharp edges and grayscale. So this slope that you see is because we want smooth transitions. If you want to do an extrusion based on an image, there are lots of software that can do it for you (such as [eXtrudy](http://www.extrudy.net/)

I think it can be a good improvement for Cura, but not possible right now. I think adding a check button asking whether is a sharp image or not, and change the implementation a bit to adjust this behavior can do the trick. 

At this moment, we don't have time to work on it in the near future, but I will keep it open in case somebody wants to implement it.
